### PR TITLE
Upgrade Issues to Issue Forms

### PR DESCRIPTION
**Pull request by PencilNavigator**
Demo: https://github.com/PencilNavigator/AtlasOS/issues/new/choose

Issue forms can decrease false issues like [ this ](https://github.com/Atlas-OS/Atlas/issues/358).
Disabled blank issues to avoid support questions like [ this ](https://github.com/Atlas-OS/Atlas/issues/329).
Added a link to Github Discussions in case someone does not have discord to ask genenral questions like [ this ](https://github.com/Atlas-OS/Atlas/issues/353#issue-1474879541) person.
Added a COD Modern Warfare choice to avoid duplicate COD Modern Warfare issues like [ this ](https://github.com/Atlas-OS/Atlas/issues/323).

Issue forms docs: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

### CODE REVIEW

**Comment by JayXTQ** [comment]
I’ll probably write a discussion later instead of that Modern Warfare 2 thing that uncognic wrote, as it requires you to have Discord to download. I’ll link it here when redone. But for now, I like it!

**Comment by SyrmGIT** [comment]
Awaiting @JayXTQ before merging.

**Comment by JayXTQ** [comment]
https://github.com/Atlas-OS/Atlas/discussions/362 there is the post. thank you for waiting, I will approve when link is changed

File: .github/ISSUE_TEMPLATE/config.yml
Comment (JayXTQ) [inline .github/ISSUE_TEMPLATE/config.yml:10]: Link here should be replaced with https://github.com/Atlas-OS/Atlas/discussions/362 as it requires Discord in the original message.

**Review by JayXTQ — CHANGES_REQUESTED** [review]
Check my comment

File: .github/ISSUE_TEMPLATE/config.yml
Comment (PencilNavigator) [reply .github/ISSUE_TEMPLATE/config.yml]: Changed.

**Review by JayXTQ — APPROVED** [review]

**Comment by PencilNavigator** [comment]
Sorry to interrupt, but i just found a mistake on the file bug_report.yaml, i used capital letters for the bug label which cause the bug label to not appear.
I submitted a new PR that will fix this. https://github.com/Atlas-OS/Atlas/pull/364

**Pull request merged**
